packages/edit-site/src/components/template-actions/index.js

```diff
@@ -124,11 +124,7 @@ function DeleteMenuItem( { onRemove, isTemplate } ) {
 	const [ isModalOpen, setIsModalOpen ] = useState( false );
 	return (
 		<>
-			<MenuItem
-				isDestructive
-				isTertiary
-				onClick={ () => setIsModalOpen( true ) }
-			>
+			<MenuItem isDestructive onClick={ () => setIsModalOpen( true ) }>
```

Comment:
This button had the tertiary variation, but was removed as it is no longer needed.

![delete-template](https://github.com/WordPress/gutenberg/assets/54422211/26b60b12-1666-4d64-927c-96f9594dd872)